EnglishTranslation-master/ETC.tsv

```diff
@@ -19679,12 +19679,12 @@ ETC_20151224_019679	The possible time to declare the war
 ETC_20151224_019680	Guild Event (Coming Soon)
 ETC_20151224_019681	Only available on channel 1.
 ETC_20151224_019682	You cannot create it nearby other objects.
-ETC_20151224_019683	Only the Guild Master is able to use it.
+ETC_20151224_019683	Only the guild master is able to use it.
 ETC_20151224_019684	Enter Game
 ETC_20151224_019685	Completed matching. You may enter the {Name} game.
 ETC_20151224_019686	The allies have failed to protect the Goddess Statue in round {Round}.
 ETC_20151224_019687	The enemies have failed to protect the Goddess Statue in round {Round}.
-ETC_20151224_019688	Guild {GuildName} has won.
+ETC_20151224_019688	The guild {GuildName} has won.
```

Comment:
I do not know the context behind this other than this line. Mentioning 'guild' makes it clear for everyone that it is about a guild rather than something else.